src/internal/m365/collection/site/backup.go

```diff
@@ -171,6 +172,8 @@ func CollectLists(
 
 	ctx = clues.Add(ctx, "can_use_previous_backup", canUsePreviousBackup)
 
+	tombstones := makeTombstones(dps)
```

Comment:
nit: This could be moved to `var` section

Reply:
this is taken care in the next PR. Here it is not possible as it is dependent on `dps`